src/main/java/net/minecraftforge/installer/actions/FatInstallerAction.java

```diff
@@ -85,6 +85,7 @@ public boolean run(File target, Predicate<String> optionals, File installer) thr
                     libraries.addAll(Arrays.asList(version.getLibraries()));
                 }
                 if (OPTIONS.contains(Options.INSTALLER_LIBS)) {
+                    libraries.addAll(Arrays.asList(profile.getLibraries()));
```

Comment:
Ah, did I forget about these? Are these the Neo libs? If so, they should be bundled as part of MC_LIBS not INSTALLER_LIBS (this one is reserved for the processors)